src/common/providers/identity.ts

```diff
@@ -835,9 +835,13 @@ export function wrapHandler(
         'skipTokenVerification'
       )
         ? unsafeDecodeAuthBlockingToken(req.body.data.jwt)
+        : handler.length === 2
+        ? await apps()
+            .admin.auth()
+            ._verifyAuthBlockingToken(req.body.data.jwt)
         : await apps()
             .admin.auth()
-            ._verifyAuthBlockingToken(req.body.data.jwt);
+            ._verifyAuthBlockingToken(req.body.data.jwt, 'run.app');
```

Comment:
The second argument to 

https://github.com/firebase/firebase-admin-node/blob/6b9760797c801e4f2e23c0e2fd4c595ae302f107/src/auth/token-verifier.ts#L414-L417  seems to be `isEmulator` and not `audience`. Did this change in some recent version of the admin sdk?

Reply:
huh that's confusing. Do you know if the difference is between `getAuth` vs `admin.auth()`?

Reply:
I believe they both return the `Auth` class, I'm pretty sure `token-verifier` is not directly exported